client/web/compose/src/components/PageBlocks/Wrap/Plain.vue

```diff
@@ -4,6 +4,19 @@
       class="d-flex flex-column position-relative h-100 border-0"
       :class="blockClass"
     >
+      <b-button
```

Comment:
Look at line 43 in the same file and do it like that please, since its pretty much the same kind of function.

And same in plain